schema/spec.md

```diff
@@ -99,132 +290,33 @@ this form:
-`compatibility` attribute. Any breaking change MUST result in a new **schema**
-Resource being created.
-
-In terms of versioning, you can think of a **schema** as a collection of
-versions that are compatible according to the selected `compatibility` mode.
-When that compatibility is broken across versions, a completely new **schema**
-MUST be created, to indicate the breaking change.
-
-## Schema Registry Model
-
-The formal xRegistry extension model of the Schema Registry resides in the
-[model.json](model.json) file.
-
-#### Schema Group
-
-A schema Group is a container for schemas that are related to each other in
-some application-defined way. This specification does not impose any
-restrictions on what schemas can be contained in a schema Group.
-
-## Schema Registry
+## 4. Schema Registry
 
 The Schema Registry is a metadata store for organizing schemas and schema
 Versions of any kind; it is a document store.
 
-The xRegistry API extension model of the Schema Registry, which is defined in
-detail below, is as follows:
-
-```yaml
-{
-  "groups": [
-    {
-      "singular": "schemagroup",
-      "plural": "schemagroups",
-
-      "resources": [
-        {
-          "singular": "schema",
-          "plural": "schemas",
-          "versions": 0,
-
-          "attributes": {
-            "format": {
-              "name": "format",
-              "type": "string",
-              "description": "Schema format identifier for this schema version"
-            }
-          },
-          "metaattributes": {
-            "validation": {
-              "name": "validation",
-              "type": "boolean",
-              "description": "Verify compliance with specified 'format'"
-            }
-          }
-        }
-      ]
-    }
-  ]
-}
-```
-
 Implementations of this specification MAY include additional extension
 attributes, including the `*` attribute of type `any`.
 
 Since the Schema Registry is an application of the xRegistry specification, all
 attributes for Groups, Resources, and Resource Version objects are inherited
 from there.
 
-### Schema Groups
+### 4.1. Schema Groups
 
 The Group (`<GROUP>`) name for the Schema Registry is `schemagroups`. The
-Group does not have any specific extension attributes.
+Schema Group does not have any specific extension attributes.
+
+A schema group is a collection of schemas that are related to each other in
+some application-defined way. A Schema Group does not impose any restrictions
+on the contained schemas, meaning that a Schema Group MAY contain schemas of
+different formats.
 
-A schema Group is a collection of schemas that are related to each other in
-some application-defined way. A schema Group does not impose any restrictions
-on the contained schemas, meaning that a schema Group MAY contain schemas of
-different formats. Every schema MUST reside inside a schema Group.
+Every schema MUST reside inside a Schema Group.
```

Comment:
It might be me, but trying to read this as a newbie someone may think that a schema can be part of a Group directly. So, perhaps a tiny bit of clarity:
`Every schema (i.e. schema Resource) MUST reside inside a Schema Group`